### ISSUE TITLE
spreads break if closing > on new line

### ISSUE THREAD

**Issue by KyleAMathews** [jsdf/coffee-react-transform#46]
This works:

```coffeescript
      <Hero
        url="something.jpg"
        slogan="yo"
        {...@props} />
```

But this doesn't
```coffeescript
      <Hero
        url="something.jpg"
        slogan="yo"
        {...@props}
      />
```

**Comment by jsdf** [comment]
Fixed in v3.0.1

**Comment by KyleAMathews** [comment]
Thanks!
